src/data/nav/chat.ts

```diff
@@ -18,8 +18,12 @@ export default {
       ],
     },
     {
-      name: 'Get started',
+      name: 'Getting started',
       pages: [
+        {
```

Comment:
Is it worth nesting the languages under a general "getting started" tab, e.g. getting started -> JS, getting started -> Kotlin

Reply:
The heading of the section is already 'Getting started', so if we do that then it becomes a double heading. We could nest them under "something" until we have 4 or so, and then move the other items out of 'Getting started'. Maybe even 'quickstart' in the interim?

Reply:
Yeah that sounds good, it's just bit weird to have the language-specific guides at the same level as more generic pages like "rooms"

Reply:
I think this looks better in the interim, but let me know what you think: 

<img width="178" alt="Screenshot 2025-04-10 at 13 03 15" src="https://github.com/user-attachments/assets/d5152f00-d616-4820-8bc0-574ccac90605" />

Reply:
Like it!